### ISSUE TITLE
Trouble Giving Access

### ISSUE THREAD

**Issue by samagnew** [documize/community#141]
Added a new user and gave him "view" access to an existing space with 100 documents. He sees the space but not the documents. This is 1.60.0-ee. The groups feature is new so I made a group and tried giving him read access this way. It didn't work either. In the older versions you could set access control at document level (had to, actually). I wonder if it is the case that these access control lists at the document level still exist and are denying him.

**Comment by HarveyKandola** [comment]
Can you upgrade to v1.61.0-ee and test again please?

Access should always be granted at either the Space and Category level.

**Comment by samagnew** [comment]
I have done the upgrade. Still no access to the space. Tried providing all access and this didn't change the behaviour. I then removed the account and used the "Invite" email link. When the user clicks on the link documize seems to load a blank page. No login window. No content. Safari, Chrome and Firefox tried. In the logs I don't see anything after the creation of the invite link:

my-user@my.org is sharing space Our Docs with existing user target-user@my.org

Sam
